mindsdb/integrations/handlers/sentence_transformer_handler/requirements.txt

```diff
@@ -0,0 +1,2 @@
+sentence-transformers
+torch
```

Comment:
`pydantic` is missing

Reply:
Same with `pandas`, but OK as it's required by MindsDB core.

Reply:
yes I left out pandas deliberately, will add pydantic tho!